src/content/docs/logs/forward-logs/heroku-log-forwarding.mdx

```diff
@@ -127,7 +158,7 @@ The formatted result looks something like this:
    ```shell
    curl -XDELETE -H 'api-key: <varYOUR_NR_LICENSE_KEY</var>' https://log-syslog-configuration-api.service.newrelic.com/heroku-account-mappings/<herokuMappingId>?accountId=<var>YOUR_NR_ACCOUNT_ID</var>
    ```
- 4. The API returns an HTTP 204 response and the drain token mapping is deleted. 
+ 4. The API returns an HTTP 204 response and the drain token mapping is deleted.
```

Comment:
@danybmx this isn't technically a step users need to do. I'd recommend you make this into a simple one-sentence paragraph that follows the steps. It would just be a summary of process.